pages/api/openPR.js

```diff
@@ -100,7 +100,8 @@ function getSubmissionFiles(values, nomineeJSON) {
     stage,
     data,
     files = {};
-  stage = values.stage;
```

Comment:
At a quick glance of the CI's output, I believe here may be where the failing build is coming from.

Reply:
https://github.com/dpgabot/submission-digitalpublicgoods/runs/6918434347?check_suite_focus=true#step:6:9